frontend/src/metabase/components/UserPicker/UserPicker.jsx

```diff
@@ -24,15 +24,19 @@ const UserPicker = ({ value, validateValue, users, canAddItems, onChange }) => {
     : null;
 
   const options = useMemo(() => {
-    return users.map(user => ({ label: user.common_name, value: user }));
+    // `label` here isn't really used because we specify `filterOption`.
+    // Normally, `options` will be filtered by its `label` if we don't provide
+    // `filterOption` to <TokenField />.
+    return users.map(user => ({ value: user }));
   }, [users]);
 
   const idKey = useCallback(value => {
     return value.id || value.email;
   }, []);
 
   const valueRenderer = useCallback(value => {
-    return value.common_name || value.email;
+    // https://user-images.githubusercontent.com/1937582/172158045-802e7600-531d-4dd8-86f9-b4bd4dc936aa.png
+    return value.common_name;
```

Comment:
`common_name` already fall back to `email`

<img src="https://user-images.githubusercontent.com/1937582/172158045-802e7600-531d-4dd8-86f9-b4bd4dc936aa.png" />